packages/core/database/src/dialects/cockroachdb/schema-inspector.ts

```diff
@@ -0,0 +1,307 @@
+import type { Database } from '../..';
+import type { Schema, Column, Index, ForeignKey } from '../../schema/types';
+import type { SchemaInspector } from '../dialect';
+
+interface RawTable {
+  table_name: string;
+}
+
+interface RawColumn {
+  data_type: string;
+  column_name: string;
+  character_maximum_length: number;
+  column_default: string;
+  is_nullable: string;
+}
+
+interface RawIndex {
+  indexrelid: string;
+  index_name: string;
+  column_name: string;
+  is_unique: boolean;
+  is_primary: boolean;
```

Comment:
Same for these removing the `index` prefix may make the variable names more clear